Description: Configfile
 TODO: Put a short summary on the line above and replace this paragraph
 with a longer explanation of this change. Complete the meta-information
 with other relevant fields (see below for details). To make it easier, the
 information below has been extracted from the changelog. Adjust it or drop
 it.
 .
 piradio (3.13-0ts-1) unstable; urgency=low
 .
   * Initial release.
Author: Tobias Schlemmer <keinstein@users.sourceforge.net>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- piradio-3.13-0ts.orig/piradio.conf
+++ piradio-3.13-0ts/piradio.conf
@@ -1,3 +1,4 @@
+[Dircetories]
 tempdir            = '/tmp/radio_stream_files/'
 playlistsdir       = libdir + '/playlists/'
 radiodir           = libdir + '/radio/'
@@ -16,7 +17,8 @@ mpdportfile        = libdir + "/mpdport"
 sharefile          = libdir + "/share"
 pidfile            = '/run/piradio.pid'
 
-# logging standard io
+[Logging]
+logfile            = '/var/log/radio.log'
 stdin              = '/dev/null'
 stdout             = '/dev/null'
 stderr             = '/dev/null'
\ No newline at end of file
--- piradio-3.13-0ts.orig/config.py
+++ piradio-3.13-0ts/config.py
@@ -1,23 +1,20 @@
-package      = 'piradio'
-libdir       = '/var/lib/'+package
-srcdir       = '/usr/share/'+package
-playlistsdir = libdir + '/playlists/'
-radiodir     = libdir + '/radio/'
-stationlist  = radiodir + 'stationlist'
-distfile     = srcdir + 'station.urls'
-tempdir      = '/tmp/radio_stream_files/'
-podcastsfile = libdir + '/podcasts'
-CurrentStationFile = libdir + "/current_station"
-CurrentTrackFile = libdir + "/current_track"
-VolumeFile = libdir + "/volume"
-TimerFile = libdir + "/timer"
-AlarmFile = libdir + "/alarm"
-StreamFile = libdir + "/streaming"
-LogLevelFile=libdir + "/loglevel"
-RssFile = libdir  + "/rss"
-MpdPortFile = libdir + "/mpdport"
+import ConfigParser
 
-input "/etc/" + package + "/piradio.conf"
-# System files
+package       = 'piradio'
+libdir        = '/var/lib/'+package
+srcdir        = '/usr/share/'+package
+configfile    = '/etc/'+package+'/'+package+'.conf'
+libconfigfile = libdir+'/'+package+'.conf'
 
 
+def load_config(filename):
+    retval = ConifgParser.ConfigParser()
+    retval.read(configfile)
+    return retval
+
+def save_config(config,filename):
+    cfgfile = open(filename,'w')
+    config.write(cfgfile)
+
+config    = load_config(configfile)
+libconfig = load_config(libconfigfile)
